src/antsibull_changelog/cli.py

```diff
@@ -486,6 +486,12 @@ def _get_version_and_codename(paths: PathsConfig, config: ChangelogConfig,
     codename: str | None = args.codename
 
     if not config.is_collection and not config.is_other_project and not (version and codename):
+        if version or codename:
+            # Only one is supplied: error out instead of simply ignoring that one
+            raise ChangelogError(  # pylint: disable=raise-missing-from
```

Comment:
Is this noqa comment actually necessary? I got no errors when I removed it and ran pylint locally.

Reply:
Ah, probably not, I copy'n'pasted it from below.

Is there a way to make pylint report such things?

```suggestion
            raise ChangelogError(
```

Reply:
> Is there a way to make pylint report such things?

It seems there is: https://pylint.readthedocs.io/en/latest/user_guide/messages/information/useless-suppression.html